### DISCUSSION TITLE
No translation at Gherkin?

### DISCUSSION THREAD

**Discussion by lucky7104** [codeceptjs/CodeceptJS#3564] [General]
Hello,
Is it possible that there is no translation for Gherkin?

I have entered the translated words in the translation file, but they are not recognized.

I looked at the part in the code and I think it really seems to be not integrated.
I think it is a pity, because especially Gherkin lives from the language.

**Comment by kobenguyent** [comment]
@lucky7104 I think the current version supports it https://codecept.io/changelog/#_3-5-7
